src/main/java/com/coveros/selenified/services/Response.java

```diff
@@ -146,14 +147,15 @@ public void assertEquals(JsonObject expectedJson) {
      * @param expectedArray - the expected response json array
      */
     public void assertEquals(JsonArray expectedArray) {
-        Success success = Success.FAIL;
+        boolean pass = false;
         if (array != null) {
-            success = array.equals(expectedArray) ? Success.PASS : Success.FAIL;
+            pass = array.equals(expectedArray) ? true : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1125)